test/interval.jl

```diff
@@ -125,12 +125,43 @@ isinf(::TimeType) = false
     end
 
     @testset "accessors" begin
-        for (a, b, _) in test_values
+        accessor_specific_vals = [
+            # test adding eps()
+            (-10.0, 10.0, nothing),
+            (-10.0, 10.0, eps(Float64)),
+
+            ('c', 'd', 2),
+            (Date(2004, 2, 13), Date(2020, 3, 13), Hour(1)),
+        ]
+        for (a, b, p) in append!(accessor_specific_vals, test_values)
             for (L, R) in BOUND_PERMUTATIONS
                 interval = Interval{L, R}(a, b)
 
                 @test first(interval) == a
                 @test last(interval) == b
+
+                # If p=eps(Float64) then we treat it as if p=nothing.
+                # This is because adding eps to floats is not accurate and we must use nextfloat and prevfloat.
+                p = p == eps() ? nothing : p
```

Comment:
Doesn't this result in `(-10.0, 10.0, nothing)` and `(-10.0, 10.0, eps(Float64))` being identical tests?

Reply:
Whooops, this is old code that I accidentally pushed. Will resolve.